WooCommerce/Classes/ViewRelated/Custom Fields/CustomFieldsListViewModel.swift

```diff
@@ -1,29 +1,49 @@
 import Combine
 import Foundation
+import Networking
+import Yosemite
 
 final class CustomFieldsListViewModel: ObservableObject {
-    private let originalCustomFields: [CustomFieldViewModel]
-
-    var shouldShowErrorState: Bool {
-        savingError != nil
-    }
+    private let stores: StoresManager
+    private var originalCustomFields: [CustomFieldViewModel]
+    private let customFieldsType: MetaDataType
+    private let siteId: Int64
+    private let parentItemId: Int64
```

Comment:
Nit: In Woo iOS we tend to use the uppercase `ID` for properties like these:

```suggestion
    private let siteID: Int64
    private let parentItemID: Int64
```